images/linux/scripts/installers/java-tools.sh

```diff
@@ -85,6 +90,22 @@ for cmd in gradle java javac mvn ant; do
     fi
 done
 
+javaVersion=`java -version 2>&1 | head -n 1 | cut -d\" -f 2`
+if [[ "$javaVersion" =~ ([1]{0,1}.)?$DEFAULT_JDK_VERSION.* ]]; then
+    echo "Java is equal to defaul"
+else
+    echo "Java is not equal to default"
+    exit 1
+fi
+
+javacVersion=`javac -version 2>&1 | sed 's/javac //g'`
```

Comment:
Why don't we use simple cut statement?
`javac -version |& cut -d" " -f2`

Reply:
Done.